AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/wrapper_qt_variable_manager.py

```diff
@@ -0,0 +1,52 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Object to house all the Qt Objects used when testing and manipulating the O3DE UI
+"""
+from editor_python_test_tools.utils import TestHelper as helper
+from PySide2 import QtWidgets, QtCore
+import pyside_utils
+from scripting_utils.scripting_constants import (VARIABLE_MANAGER_QT, VARIABLE_TYPES, VARIABLE_PALETTE_QT,
+                                                 ADD_BUTTON_QT, WAIT_TIME_3, GRAPH_VARIABLES_QT)
+from editor_python_test_tools.utils import Report
+
+class WrapperQTVariableManager:
```

Comment:
Since there's more than one Variable Manager in the O3DE Editor, let's update the name to reflect that this is the SC Variable Manager.